v2/dtos/command.go

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+import "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+
+// Command defines a specific read/write operation targeting a device
+// This object and its properties correspond to the Command object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/Command
+type Command struct {
+	Id   string        `json:"id" yaml:"id,omitempty"`     // Id is a unique identifier, such as a UUID
+	Name string        `json:"name" yaml:"name,omitempty"` // Command name (unique on the profile)
+	Get  CommandAction `json:"get" yaml:"get,omitempty"`   // Get Command
+	Put  CommandAction `json:"put" yaml:"put,omitempty"`   // Put Command
```

Comment:
In today's Core WG it was decide that this should be something simple to indicate if the Get and/or Put are present. How about making them a boolean?
```
	Get  bool `json:"get" yaml:"get,omitempty"`   // Get Command enabled
	Put  bool `json:"put" yaml:"put,omitempty"`   // Put Command enabled
```

Reply:
+1

Reply:
Replaced the CommandAction with boolean and updated the Swagger file at edgexfoundry/edgex-go/pull/2627.